torquedata/core/views.py

```diff
@@ -183,22 +243,29 @@ def get_row(group, wiki_key, key, fmt, sheet_name, view=None):
     elif fmt == "dict":
         return row
     else:
-        raise Exception(f"Invalid format {fmt}")
+        raise Exception("Invalid format {fmt}")
```

Comment:
this should be kept as an f-string, right?

Reply:
Oh good call!!  Whoops